README.md

```diff
@@ -0,0 +1,8 @@
+The signtool is an utility that can be used to issue sign/verify requests for a development or
+production mode.
+
+The development mode uses keys generated from an OpenSSL environment, while a production mode
+uses the IBM Crypto Card to generate and sign builds.
```

Comment:
Is a *requirement* to use an IBM crypto card? Or is that just an implementation detail of how IBM will do things?